### ISSUE TITLE
False positive `no-array-prototype-extensions` for emberArray combined with `this`

### ISSUE THREAD

**Issue by lin-ll** [ember-cli/eslint-plugin-ember#1807]
```
import { A } from '@ember/array';

this.foo = A();
this.foo.toArray();
```
